src/interfaces/routes.ts

```diff
@@ -220,6 +220,24 @@ export interface GetUserTradesResponse {
   tradeType: string;
 }
 
+export interface GetUserTradesHistoryRequest {
+  symbol?: MarketSymbol;
+  maker?: boolean;
+  fromId?: number;
+  startTime?: number;
+  endTime?: number;
+  limit?: number;
+  cursor?: number;
+  type?: ORDER_TYPE;
+  parentAddress?: string;
+}
+
+export interface GetUserTradesHistoryResponse {
+  data: GetUserTradesResponse[];
+  nextCursor: number;
```

Comment:
We do not take the cursor in the request. Is pagination implemented in the userTradesHistory API. I am thinking if we do not have pagination then this property should not be exposed to the clients

Reply:
![image](https://github.com/fireflyprotocol/bluefin-v2-client-ts/assets/42298472/3c3f456a-f3ce-49ed-bb4a-fe07dae9df00)